templates/docs/document.html

```diff
@@ -18,68 +52,234 @@
-          {{ document.body_html | safe }}
-        </div>
-      </div>
+  <section class="p-strip u-extra-space">
+    <div class="row">
+      <aside class="col-3">
+        {% if versions | length > 1 %}
+        <label for="version-select" class="u-hide">Version</label>
+        <select name="version-select" id="version-select" onChange="window.location.href=this.value">
+        {% for version in versions %}
+          {% set active = docs_version == version['path'] %}
+          <option value="{{ version_paths[version['path']] }}"{% if active %} selected{% endif %}>{{ version['version'] }}</option>
+        {% endfor %}
+        <select>
+        {% endif %}
 
-      <div class="p-strip is-shallow u-no-padding--bottom">
-        <div class="l-docs-row row">
-          <hr />
-          <i>Last updated {{ document.updated }}.</i>
+        <nav data-js="navigation" class="p-side-navigation" id="{{ navigation['path'] or 'default' }}">
+        <a href="#{{ navigation['path'] or 'default' }}" class="p-side-navigation__toggle js-drawer-toggle" aria-controls="{{ navigation['path'] or 'default' }}">
+          Toggle side navigation
+        </a>
+        <div class="p-side-navigation__overlay js-drawer-toggle" aria-controls="{{ navigation['path'] or 'default' }}"></div>
+        <div class="p-side-navigation__drawer">
+          <div class="p-side-navigation__drawer-header">
+            <a href="#" class="p-side-navigation__toggle--in-drawer js-drawer-toggle" aria-controls="{{ navigation['path'] or 'default' }}">
+              Toggle side navigation
+            </a>
+          </div>
+          {% for nav_group in navigation.nav_items %}
+          {% if not nav_group.hidden %}
+            {% if nav_group.navlink_text %}
+              {% if nav_group.navlink_href %}
+              <h3 class="p-side-navigation__heading--linked">
+                <a class="p-side-navigation__link" href="{{ nav_group.navlink_href }}" {% if nav_group.is_active %}aria-current="page"{% endif %}>
+                  {{ nav_group.navlink_text }}
+                </a>
+              </h3>
+              {% else %}
+                <h3 class="p-side-navigation__heading">{{ nav_group.navlink_text }}</h3>
+              {% endif %}
+            {% endif %}
+            {#
+              Use `create_navigation(nav_group.children)` for a default, fully expanded navigation.
+              Use `create_navigation(nav_group.children, expandable=True)` for the nested nav levels to expand only when parent page is active.
+            #}
+            {{ create_navigation(nav_group.children, expandable=True) }}
+          {% endif %}
+        {% endfor %}
         </div>
-      </div>
-      <div class="p-strip is-shallow u-no-padding--bottom u-sticky--bottom">
-        <div class="l-docs-row row">
-          <div class="p-notification--information">
-            <p class="p-notification__content">
+      </nav>
+      </aside>
+      <main class="col-9" id="main-content">
```

Comment:
we already have `main#main-content` on the page, IDs needs to be unique.

```suggestion
      <main class="col-9" id="content-section">
```